user-data-store-server/src/test/java/com/wultra/security/userdatastore/restclient/DocumentRestClientTest.java

```diff
@@ -84,6 +84,22 @@ void testPostComposite() throws Exception {
         assertEquals(1, response.attachments().size());
     }
 
+    @Test
+    void testDeleteComposite() throws Exception {
+        List<EmbeddedPhotoCreateRequest> photos = new ArrayList<>();
+        photos.add(new EmbeddedPhotoCreateRequest("test_type", "test_data", null));
```

Comment:
We may use JDK 11+ API.
```suggestion
        final List<EmbeddedPhotoCreateRequest> photos = List.of(new EmbeddedPhotoCreateRequest("test_type", "test_data", null);
```